src/cmds/pick.rs

```diff
@@ -78,9 +78,12 @@ impl Command for PickCommand {
 
         // filtering...
         // pym scripts
-        if m.is_present("plan") {
-            let ids = crate::pym::exec(m.value_of("plan").unwrap_or(""))?;
-            crate::helper::squash(&mut problems, ids)?;
+        #[cfg(feature = "pym")]
+        {
+            if m.is_present("plan") {
+                let ids = crate::pym::exec(m.value_of("plan").unwrap_or(""))?;
+                crate::helper::squash(&mut problems, ids)?;
+            }
         }
```

Comment:
I didn't implement this feature just because I don't know we can derive feature like this...